addons/udes_common/models/models.py

```diff
@@ -134,3 +135,46 @@ def selection_display_name(self, selection_field_name):
     # Odoo has a nice function that handles all the weird ways you can define selections!
     selection_values_dict = dict(selection_field._description_selection(self.env))
     return selection_values_dict.get(self[selection_field_name], False)
+
+
+class Base(models.AbstractModel):
+    _inherit = "base"
+
+    # Adding a default empty list for base model, it will be override when needed for
+    # specific models.
+    HiddenFieldsFromFormView = []
```

Comment:
Suggestion: Not related to your recent changes, but this variable name is a bit long and doesn't indicate that the fields are only hidden by default (although the comment does explain this of course).

Maybe `DetailedFormViewFields` ?

Not a huge deal at all, just something to consider. :)